common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -334,6 +334,57 @@
 
                 });
 
+                @for(membershipAccess <- item.membershipAccess if conf.Switches.MembersAreaSwitch.isSwitchedOn) {
```

Comment:
I would expect a test to assert this does what we expect.

An accident here could put our site behind a paywall.